media/css/style.css

```diff
@@ -5,60 +5,65 @@
 
 @charset "utf-8";
 
+@import url('https://rsms.me/inter/inter.css');
```

Comment:
Instead of importing a remote stylesheet, can we add the font files and rules to this repository instead?